test/calendar.test.tsx

```diff
@@ -9,14 +9,16 @@ const CalendarWithUtils = getCalendar(new DateFnsUtils());
 
 export default test("Calendar")
   .step(Page.visit("/"))
-  .child("filter by locator", (test) => test.step(renderComponent()).assertion(Calendar("18 August 2014").exists()))
-  .child("filter by title", (test) =>
-    test.step(renderComponent()).assertion(Calendar({ title: "August 2014" }).exists())
+  .child("filters", (test) =>
+    test
+      .step(renderComponent())
```

Comment:
Since all the of the steps in this test use the same render, maybe should make the individual actions children of the single parent that does the `renderComponent()` ?